log/mod.ts

```diff
@@ -431,15 +431,23 @@ const state = {
  * For examples check source code of {@linkcode FileHandler}`
  * and {@linkcode TestHandler}.
  */
-export const handlers = {
+export const handlers: {
```

Comment:
Actually, going to revert this. Deno doc understands it so I should be able to get the other thing to as well.